.travis.yml

```diff
@@ -18,10 +18,13 @@ addons:
     apt:
         sources:
             - ubuntu-toolchain-r-test
+            - kalaris-cmake
         packages:
             - swig
             - valgrind
             - g++-4.8
+            - cmake
+            - python3-dev
```

Comment:
Please update the comments above to explain why we are installing (upgrading, in fact) these two packages.